packages/appstore/src/constants/tour-steps-config.tsx

```diff
@@ -1,80 +1,111 @@
 import { Step, Styles, Locale } from 'react-joyride';
 import React from 'react';
-import { Text, SpanButton } from '@deriv/components';
-import { Localize, localize } from '@deriv/translations';
+import { Text, SpanButton, Icon } from '@deriv/components';
+import { Localize } from '@deriv/translations';
 import 'Components/toggle-account-type/toggle-account-type.scss';
+import { isMobile } from '@deriv/shared';
 
-export const tour_step_config: Step[] = [
+export const getTourStepConfig = (): Step[] => [
```

Comment:
Do we need this to be a function? Since Localize component is used, I think u can make it as a constant?

Reply:
its working as per your suggestion. i will change it to variables 👍🏼